tasks/clamav-scan.yaml

```diff
@@ -0,0 +1,89 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: clamav-scan
+spec:
+  params:
+    - name: image-digest
+      description: Image digest to scan
+    - name: image-url
+      description: Url to image
+  steps:
+    - name: extract-and-scan-image
+      image: quay.io/redhat-appstudio/hacbs-test:v1.0.3
+      workingDir: /workspace/clamav
+      script: |
+        # copy virus database from sidecar
+        cp -r /tmp/clamdb/* /var/lib/clamav
+        imagewithouttag=$(echo '$(params.image-url)' | sed "s/\(.*\):.*/\1/" | tr -d '\n')
+
+        # strip new-line escape symbol from parameter and save it to variable
+        imageanddigest=$(echo $imagewithouttag@'$(params.image-digest)')
+
+        oc image extract -a /workspace/registry-auth/.dockerconfigjson $imageanddigest
```

Comment:
Does not work with images built by s2i
```
+ oc image extract -a /workspace/registry-auth/.dockerconfigjson quay.io/mkovarik/single-nodejs-app@sha256:eba023bd2ea9634b4b19fb1bee4043d14431be713484d4f58bb0dd3ab8745753
error: unable to extract layer sha256:1a6ae3d35ced1c7654b3bf1a66b8a513d2ee7f497728e0c5c74841807c4b8e77 from quay.io/mkovarik/single-nodejs-app@sha256:eba023bd2ea9634b4b19fb1bee4043d14431be713484d4f58bb0dd3ab8745753: operation not permitted

step-modify-clam-output-to-json
2022/06/07 07:59:57 Skipping step because a previous step failed

sidecar-database
```